core/src/main/java/org/apache/calcite/util/format/FormatModels.java

```diff
@@ -55,6 +56,8 @@
 import static org.apache.calcite.util.format.FormatElementEnum.WW;
 import static org.apache.calcite.util.format.FormatElementEnum.YY;
 import static org.apache.calcite.util.format.FormatElementEnum.YYYY;
+import static org.apache.calcite.util.format.FormatElementEnum.day;
+import static org.apache.calcite.util.format.FormatElementEnum.dy;
```

Comment:
The imports here should be in alphabetical order.

Reply:
I tried it, but it will report an error. It seems that there is a restriction on the upper and lower case order.
import static org.apache.calcite.util.format.FormatElementEnum.*; ?

Reply:
Wouldn't checkstyle complain if the imports are in the wrong order?

Reply:
Possibly checkstyle is using a different collation (e.g. different treatment of upper/lower case, spaces, underscores). Do whatever makes checkstyle happy.

Reply:
Easiest thing is to make the enum constants upper-case.